docker-compose.yml

```diff
@@ -29,6 +29,7 @@ services:
         - ols-solr
         - ols-mongo
       environment:
+        - JAVA_OPTS=-Xms2g -Xmx4g -XX:+UseG1GC -XX:+HeapDumpOnOutOfMemoryError
```

Comment:
@kevinschaper were these the latest params that did work? As I'm about the approve, I recall now that you tried some commands that did not work.

Reply:
ok, were you not pointing to this PR as one that should be merged?

Reply:
closing since this seems outdated then